src/main/java/org/flickit/flickitassessmentcore/adapter/out/persistence/evidence/EvidenceMapper.java

```diff
@@ -16,7 +16,8 @@ public static EvidenceJpaEntity mapCreateParamToJpaEntity(CreateEvidencePort.Par
             param.lastModificationTime(),
             param.createdById(),
             param.assessmentId(),
-            param.questionId()
+            param.questionId(),
+            0L
```

Comment:
میتونیم 0L رو به صورت Constant تعریف کنیم.
کاری که برای DeleteAssessment انجام شد این بود:
تو لایه سرویس، یه پکیج constant ساختیم.
داخل اون پکیج یک کلاس AssessmentConstants رو به صورت زیر تعریف کردیم:

```
public final class AssessmentConstants {
    public static final long NOT_DELETED_DELETION_TIME = 0L;

    private AssessmentConstants() {
        // Private constructor to prevent instantiation
    }
}
```